.github/workflows/test_dbt_fal.yml

```diff
@@ -0,0 +1,167 @@
+name: Test dbt-fal adapter
+
+on:
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
+    paths-ignore:
+      - "examples/**"
+      - "docsite/**"
+      - "README.md"
+      - "LICENSE"
+      - "docker/**"
+
+  push:
+    branches: [act1]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        profile: ["postgres", "bigquery", "snowflake", "duckdb"]
+        python: ["3.8"]
+        environment: ["local", "venv", "conda"]
+        # bigquery implementation fails due to concurrency, let's
+        # limit it to 1 for now.
```

Comment:
Thinking about it: this probably will fail for snowflake too. Want to limit it?

Reply:
It works for now, not sure what's going on but we can limit.

Reply:
I've tried without any concurrency, let's see whether it would work now (instead of running venv, local, conda parallel; now running them sequentially one by one in the same job)